src/common/reducers/entities/repository.js

```diff
@@ -23,7 +23,14 @@ export default function repository(state={
         isSelected: true
       };
     }
-    case ActionTypes.REPO_SUCCESS:
+    case ActionTypes.REPO_SUCCESS: {
+      return {
+        ...state,
+        isSelected: true,
```

Comment:
I would say that `isSelected` doesn't need to be set here, as this action doesn't have anything to do with selection.